.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "Documentation"
+about: Report an issue related to the NumPy documentation
+
+---
+
+## Documentation
+
+<!-- If this is an issue with the current documentation for NumPy (i.e.
+incomplete/innacurate docstring, unclear explanation in any part of the
+documentation), make sure to leave a reference to the document/code you're
+referring to. -->
```

Comment:
We should also tell people to check the corresponding devdocs page (https://numpy.org/devdocs/) too, in case they are looking at an old url